consensus/src/experimental/buffer_item.rs

```diff
@@ -92,12 +102,13 @@ impl BufferItem {
         author: Author,
         signature: Ed25519Signature,
         verifier: &ValidatorVerifier,
-    ) -> Self {
+    ) -> (Self, CommitVote) {
```

Comment:
why not put the vote in the buffer item?

Reply:
sounds good. We need to retry sending commit votes in the main loop.

Reply:
the main loop can grab the vote from the Item?